apps/aecore/src/aec_blocks.erl

```diff
@@ -1,7 +1,10 @@
+%%% -*- erlang-indent-level:4; indent-tabs-mode: nil -*-
```

Comment:
We may consider slowly moving away from these using [editorconfig](https://github.com/aeternity/epoch/blob/78269fd936122b74fd00be617320ad4758b698b0/.editorconfig#L4-L5).